src/components/ArticleCard.jsx

```diff
@@ -13,7 +15,11 @@ export default function ArticleCard(props) {
                     <Card.Text>
                         {article.author} 
                         <br/>
-                        {article.created_at}</Card.Text>
+                        {article.created_at}
+                    </Card.Text>
+                <Link to={`/articles/${article.article_id}`}>
+                    <Button variant="primary">View Article</Button>
+                </Link>
                 </Card.Body>
             </Card>
         </>    
```

Comment:
theses are unnecessary as the card is already a single parent element

Reply:
done - thanks I understand when <> are needed a lot better now